pkg/cmd/server/api/types.go

```diff
@@ -144,6 +144,24 @@ type ProjectConfig struct {
 	// It is in the format namespace/template and it is optional.
 	// If it is not specified, a default template is used.
 	ProjectRequestTemplate string
+
+	// SecurityAllocator controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled.
```

Comment:
> If the allocator is off it just means your pods will fail to start with an error "no uid".  We'll want to deal with that in the SCC changes.

Oh, is that all.  For a minute there I was worried that I wouldn't be able to do things like run pods....